### PR TITLE
enable base control through rviz interactive markers

### PR DESCRIPTION
**Pull request by younesr1**

### CODE REVIEW

**Comment by younesr1** [comment]
the actual addition of the rviz markers will be done in the nightingale project repo

**Comment by younesr1** [comment]
Tested in simulation:
image

Will test on real robot tomorrow

File: movo_desktop/movo_viz/rviz/robot.rviz [hunk 1]
```diff
@@ -69,97 +66,251 @@ Visualization Manager:
...
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+        kinect2_link:
           Alpha: 1
           Show Axes: false
           Show Trail: false
           Value: true
-        base_link:
+        kinect2_rgb_link:
```
Comment (FelixKHWong) [inline movo_desktop/movo_viz/rviz/robot.rviz:96]: I see there's a bunch of links related to the kinect. How dod you know to add all of these links?

**Review by FelixKHWong — DISMISSED** [review]
Just wondering how you knew what to add to the rviz file but otherwise looks good

**Comment by younesr1** [comment]
> Just wondering how you knew what to add to the rviz file but otherwise looks good

the rviz file isnt manually modified. u change the config through the rviz gui and then u save the file

File: movo_desktop/movo_viz/rviz/robot.rviz (hunk 1 above)
Comment (younesr1) [reply movo_desktop/movo_viz/rviz/robot.rviz:96]: i just loaded the robot model which had the kinect. didnt your realsense changes get merged in? i wouldve expected rviz to show the realsense

**Comment by younesr1** [comment]
@FelixKHWong can u reapprove? i just rebased

**Review by FelixKHWong — DISMISSED** [review]

**Review by le2tang — DISMISSED** [review]
Looks good. The ROS_MASTER_URI on movo1 will be fixed with this PR: https://github.com/nightingale-project/kinova-movo/pull/16

**Review by FelixKHWong — APPROVED** [review]

**Pull request merged**